etl/steps/data/garden/health/2023-04-18/wgm_mental_health.py

```diff
@@ -0,0 +1,339 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from typing import List
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+from .shared import (
+    MAPPING_AGE_VALUES,
+    MAPPING_COLUMN_NAMES,
+    MAPPING_GENDER_VALUES,
+    MAPPING_QUESTION_VALUES,
+)
+
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+# Minimum number of participants answering a question. Questions for demographic groups with fewer answers are filtered out.
+# This mostly affects very granular breakdowns (e.g. by age and gender)
+THRESHOLD_ANSWERS = 100
+
+
+def load_countries_regions() -> Table:
+    """Load countries-regions table from reference dataset (e.g. to map from iso codes to country names)."""
+    ds_reference: Dataset = paths.load_dependency("reference")
+    tb_countries_regions = ds_reference["countries_regions"]
+
+    return tb_countries_regions
+
+
+def load_population() -> Table:
+    """Load population table from population OMM dataset."""
+    ds_indicators: Dataset = paths.load_dependency(channel="garden", namespace="demography", short_name="population")
+    tb_population = ds_indicators["population"]
+
+    return tb_population
+
+
+def run(dest_dir: str) -> None:
+    log.info("wgm_mental_health: start")
+
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow: Dataset = paths.load_dependency("wgm_mental_health")
+
+    # Read table from meadow dataset.
+    tb_meadow = ds_meadow["wgm_mental_health"]
+
+    # Create a dataframe with data from the table.
+    df = pd.DataFrame(tb_meadow)
```

Comment:
I find it clearer to do all loadings at the beginning, in the "Load inputs" section, instead of loading additional data in the "Process data" section. So you could load here population and countries-regions, and then ingest them as arguments. But if it's too much hassle, ignore.